### PR TITLE
Marshal loadout changes, some fixes - Its a omni pr

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
## About The Pull Request
<details>
<summary>
<br>Gives sergeant account access

Fixes an incorrect path in the kneepads accessory. whoops!

made folders, clipboards and CorpRegs free in the loadout.

Marshal loadout changes. "Main" weapons now come with 2/1 lethal to LTL ammo, sidearms come 1/2 lethal to LTL.

Whirbelwind placed into the sidearm category where it belongs, in its place is the Thompson SMG.

Specialist+Ranger get main-arm vouchers now. Similarly to how Sergeant, Specs voucher includes special options not available to regular officers, Rangers get a standard.(They both retain their 'standard' sidearm in the locker.)

Pistol holster buffed, now fits; Aedes, Galaxy, Sky, Straylight, and ladon pistols. DM me if i'm missing any obvious pistols that need to be added.

Adjusts some of the ammo damage/pen values to those /actually/ listed in the previous changelog.

Fixes Ranger spawn hardcase - should properly populate its contents.

WO gun-box, Warrant officer now has a small box that allows them to select between the SWAT combat shotgun and Osprey precision rifle as their main-arm.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/game/jobs/job/security.dm [hunk 1]
```diff
@@ -187,6 +187,7 @@
 	supervisors = "the Blackshield Commander"
 	difficulty = "Hard."
 	selection_color = "#a7bbc6"
+	department_account_access = TRUE
```
Comment (Trilbyspaceclone) [inline code/game/jobs/job/security.dm:190]: `>:T`

Comment (cdb-is-not-good) [reply code/game/jobs/job/security.dm:190]: what!

**Comment by cdb-is-not-good** [comment]
> Corporate Regulations don't have anything in them.
gdi - trilby told me the opposite and i couldn't remember

File: code/game/objects/items/weapons/storage/marshal_kits.dm [hunk 2]
```diff
@@ -6,15 +6,15 @@
 	icon_state = "box"
 
 // Primary kits
-/obj/item/storage/box/m_kit/wirbelwind
-	name = "\improper Wirbelwind Kit"
-	desc = "The standard Marshal box kit containing a wirbelwind submachine gun. Capable of fitting a holster and going full auto because who needs accuracy?"
+/obj/item/storage/box/m_kit/thompson
+	name = "\improper Thompson Kit"
+	desc = "The standard Marshal box kit containing a thompson submachine gun.An SMG worthy of main-carry, as long as one doesn't mind looking a bit dated "
```
Comment (r4d6) [inline code/game/objects/items/weapons/storage/marshal_kits.dm:11]:
```suggestion
	desc = "The standard Marshal box kit containing a thompson submachine gun. An SMG worthy of main-carry, as long as one doesn't mind looking a bit dated "
```

**Review by r4d6 — COMMENTED** [review]
While I love expanding the voucher loadout stuff, why have the Marshals' kit have more lethal ammo than non-lethal ?

I know that they _can_ switch to lethal ammo if non-lethals don't work, but I don't think that's a valid reason to have 2/3rd of their loadout be lethal ammo.
- The Pug Kit has 1 beanbag magazine and 2 lethal
- The Bulldog has 1 rubber magazine and two lethal.
- Same for the Auto-Shotgun kit.
- Same for the Thompson kit.

I don't mind them having lethal ammo at all. I mind that the lethal ammo is the _majority_ of the ammo they get.

**Comment by cdb-is-not-good** [comment]
> While I love expanding the voucher loadout stuff, why have the Marshals' kit have more lethal ammo than non-lethal ?
> 
> I know that they _can_ switch to lethal ammo if non-lethals don't work, but I don't think that's a valid reason to have 2/3rd of their loadout be lethal ammo.
> 
> * The Pug Kit has 1 beanbag magazine and 2 lethal
> * The Bulldog has 1 rubber magazine and two lethal.
> * Same for the Auto-Shotgun kit.
> * Same for the Thompson kit.
> 
> I don't mind them having lethal ammo at all. I mind that the lethal ammo is the _majority_ of the ammo they get.

This is...entirely intentional. lethal ammo is also not the majority of what they get - they have 50/50 split differently between their main arm(which will mostly end up being used against bugs) and their sidearm(which is unlikely to be used at all - but is more likely to be used against criminals.). the total spread of their 6 mags is 3 lethal 3 non lethal. 

> I know that they _can_ switch to lethal ammo if non-lethals don't work, but I don't think that's a valid reason to have 2/3rd of their loadout be lethal ammo.

This line in particular i take umbrage against. This is...a fringe colony under constant threat. All perps are going to be armed, almost all perps will be armored - and they're not even the ones you have to worry about, it's the giant spiders who will be only lightly tickled by the rubber rounds.

**Comment by Trilbyspaceclone** [comment]
How does this buff the church?

**Comment by cdb-is-not-good** [comment]
> How does this buff the church?

>:T

**Comment by Kazkin** [comment]
I specifically told you I was iffy on giving them lethals buuuuuuuuut fine, there was clearly effort that was put in that should be rewarded.

**Pull request merged**